services/app-web/src/pages/StateSubmission/Contacts/Contacts.test.tsx

```diff
@@ -89,14 +89,14 @@ describe('Contacts', () => {
 
         await waitFor(() => {
             expect(
-                screen.getByText('You must provide a name')
```

Comment:
Have you used [`within`](https://testing-library.com/docs/dom-testing-library/api-within/) before? I think checking to see that the error text is repeated is a fine way to do this but want to make sure you know about `within` b/c it's useful.

Reply:
I have! I was torn about if it was worth having separate assertions against the error summary and the field error area.

Reply:
I kinda want these tests to say "at least once" and then have the specific error display and behavior tested elsewhere. But hopefully this won't change again soon, so it's probably fine as-is.